<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Nighty3098</author>
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>687</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>График</string>
  </property>
  <property name="styleSheet">
   <string notr="true">	background-color: #171b22;
	color: #fff;</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <property name="spacing">
         <number>12</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="graphBox">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0d1117; border: 0px; border-radius: 10px;</string>
          </property>
          <property name="title">
           <string>График</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="graphLayout"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="styleSheet">
             <string notr="true">background-color: #0d1117; border: 0px; border-radius: 10px;</string>
            </property>
            <property name="title">
             <string>Формула графика</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>12</number>
             </property>
             <property name="topMargin">
              <number>12</number>
             </property>
             <property name="rightMargin">
              <number>12</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QLabel" name="equationLabel">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>300</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: transparent;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>resources/equation.png</pixmap>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #0d1117; border: 0px; border-radius: 10px;</string>
            </property>
            <property name="title">
             <string>Параметры графика</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QGridLayout" name="gridLayout_3">
               <property name="leftMargin">
                <number>12</number>
               </property>
               <property name="topMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>12</number>
               </property>
               <property name="bottomMargin">
                <number>12</number>
               </property>
               <property name="horizontalSpacing">
                <number>12</number>
               </property>
               <property name="verticalSpacing">
                <number>6</number>
               </property>
               <item row="2" column="1">
                <widget class="QLineEdit" name="stepLineEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #7289DA;
selection-background-color: #999999;
selection-color: #fff;
color: #fff;
border-width: 0px;
border-style: solid;
border-radius: 10px;
border-color: #0d1117;
alternate-background-color: #181818;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="maximumSize">
                  <size>
                   <width>165</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Шаг изменения переменной (Xh)</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="endLineEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #7289DA;
selection-background-color: #999999;
selection-color: #fff;
color: #fff;
border-width: 0px;
border-style: solid;
border-radius: 10px;
border-color: #0d1117;
alternate-background-color: #181818;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Начало промежутка (Xn)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="startLineEdit">
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #7289DA;
selection-background-color: #999999;
selection-color: #fff;
color: #fff;
border-width: 0px;
border-style: solid;
border-radius: 10px;
border-color: #0d1117;
alternate-background-color: #181818;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Конец промежутка (Xk)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="startProcess">
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
	background-color: #7289DA;
	selection-background-color: #7289DA;
	color: #fff;
	border-width: 2px;
	border-style: solid;
	border-radius: 10px;
	alignment: center;
	border-color: #0d1117;
}</string>
                 </property>
                 <property name="text">
                  <string>Построить график</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
